application_store_api/namespace_fund/data_store/store.py

```diff
@@ -0,0 +1,117 @@
+import datetime
+import json
+import uuid
+
+from application_store_api.namespace_fund.data_store.data import (
+    funding_service_design_application,
+)
+from application_store_api.namespace_fund.data_store.data import (
+    initial_fund_store_state,
+)
```

Comment:
Shorter name conventions could make this simpler here perhaps?

Reply:
I'm going to have to disagree, I like long snake case names because they very clearly describe the function of a file, function, etc. 

That said, I do like using from X import Y as Z in order to make the function of something within the file very clear.

Reply:
yes i think using the 'as Z' solves both points above - thanks guys